mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -136,28 +152,22 @@ def ocn_modelvsobs(config, field):
             # Increment month value to be consistent with the model output
             dsObs.iMONTH.values += 1
 
-            # Rename the dimensions to be consistent with other obs. data sets
-            dsObs.rename({'month': 'calmonth', 'lat': 'latCoord',
-                          'lon': 'lonCoord'}, inplace=True)
-            dsObs.rename({'iMONTH': 'Time', 'iLAT': 'lat', 'iLON': 'lon'},
-                         inplace=True)
+            dsObs.rename({'iMONTH': 'Time'}, inplace=True)
             # set the coordinates now that the dimensions have the same names
-            dsObs.coords['lat'] = dsObs['latCoord']
-            dsObs.coords['lon'] = dsObs['lonCoord']
-            dsObs.coords['Time'] = dsObs['calmonth']
-            dsObs.coords['month'] = ('Time', np.array(dsObs['calmonth'], int))
+            dsObs.coords['lat'] = dsObs['lat']
+            dsObs.coords['lon'] = dsObs['lon']
+            dsObs.coords['Time'] = dsObs['month']
+            dsObs.coords['month'] = ('Time', np.array(dsObs['month'], int))
             # no meaningful year since this is already a climatology
             dsObs.coords['year'] = ('Time', np.ones(dsObs.dims['Time'], int))
 
             dsObs = mpas_xarray.subset_variables(dsObs, [obsFieldName,
                                                          'month'])
-            # Reorder dataset for consistence with other obs. data sets
-            dsObs = dsObs.transpose('Time', 'lat', 'lon')
```

Comment:
I assume @vanroekel added it because it seemed important to have the data sets in the right order, or because he was converting to numpy arrays at some point.  I think the code has evolved quite a bit since it was originally written so it's a little hard to be sure why it used to be necessary.